lib/avatax/api.rb

```diff
@@ -1,5 +1,6 @@
 require File.expand_path('../connection', __FILE__)
 require File.expand_path('../request', __FILE__)
+require 'active_support'
```

Comment:
You can remove the equivalent line in spec_helper.rb:

https://github.com/avadev/AvaTax-REST-V2-Ruby-SDK/blob/29b239ae138ae277c9d4eba2c587469bff996775/spec/spec_helper.rb#L3